inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -325,6 +327,40 @@ void prepare_primitive_fusing::fuse_simple_primitives(program_impl &p) {
             return false;
         };
 
+        auto activation_supports_fusings = [](activation_node& node) -> bool {
+            // We don't want to fuse activation when activation_kernel_opt is possible to use
```

Comment:
Have you checked the performance? Is separate nodes with opt activation kernel really faster in cases like activation + quantize?

Reply:
**Performance results:**
  | Enabled fusing   execution time | Disabled fusing   execution time | Diff
-- | -- | -- | --
Sizes: | [us] | [us] | [us]
[b:1, f:128,   x:64, y:64, z:1, w:1, g:1] | 299.83 | 313.52 | -13.69
[b:1, f:512,   x:64, y:64, z:1, w:1, g:1] | 1294.18 | 1319.57 | -25.38
[b:1, f:1024,   x:64, y:64, z:1, w:1, g:1] | 2669.42 | 2730.30 | -60.88
[b:1, f:2048,   x:64, y:64, z:1, w:1, g:1] | 5505.50 | 5599.74 | -94.24
[b:1, f:4096,   x:64, y:64, z:1, w:1, g:1] | 11703.27 | 11757.30 | -54.03
[b:1, f:4096,   x:128, y:128, z:1, w:1, g:1] | 167022.00 | 166827.67 | 194.33
[b:1, f:4096,   x:256, y:256, z:1, w:1, g:1] | 783578.00 | 766798 | 16780.00
  |   |   |  


**Topology:**
- activation -> scale -> quantize(i8) -> reorder

When it comes to big size input data, disabling activation fusing support and using activaction_kernel_opt instead seems to make sense.

Whats your opinion?

Reply:
Disabling fusing is removed.